spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonSerializer.java

```diff
@@ -134,6 +134,17 @@ public void setUseTypeMapperForKey(boolean isKey) {
 		}
 	}
 
+	/**
+	 * Configure the ObjectMapper before it is used to construct the ObjectWriter.
+	 * @param objectMapper configurable (or replaceable) ObjectMapper instance
+	 * @return configured (or replaced) ObjectMapper instance
+	 * @since 2.7.1
+	 */
+	public ObjectMapper customizeObjectMapper(ObjectMapper objectMapper) {
```

Comment:
This can be a `default` method on the interface.

Please change all sinces to 2.5.13; we will be backporting this to 2.6, 2.5.

Reply:
That was my original thought, but I took guidance from JsonTypeResolver.  If we provide default implementation, then @FunctionalInterface goes away.  Is that what you intend?

Reply:
I don't think this needs to be a `FunctionalInterface` the type resolver can be used as lambda expression, or method reference:

https://github.com/spring-projects/spring-kafka/blob/bbc5003cf7e26a3166924f06a2a407b33cf591eb/spring-kafka/src/test/java/org/springframework/kafka/support/serializer/JsonSerializationTests.java#L360

This is just a super interface for the (de)serializer.